### ISSUE TITLE
Stop tracking binaries and ipynb_checkpoints

### ISSUE THREAD

**Issue by luiarthur** [lanl/SEPIA#36]
Currently, it looks like some stuff in `.ipynb_checkpoints/` are being tracked by git.

Also, binaries like images and pickle files from demos are also being tracked. Those should ideally be removed, unless there's a reason to keep them, especially since they are already tracked in the notebooks.

Scripts converted from notebooks should also be removed, again unless there's a strong reason to track them.

@jgattiker Is it OK if we stop tracking some of these files?
- See some of the binaries and checkpoints being tracked in the diffs:
    - https://github.com/lanl/SEPIA/compare/dev

**Comment by luiarthur** [comment]
I see some of the pickle files are used to test if a model can be restored, those I think can stay.

**Comment by luiarthur** [comment]
Better yet, the pickle files could be created during the tests, instead of created beforehand. That way they don't need to be tracked.

**Comment by jgattiker** [comment]
Yeah, definitely. I'd like to have the core distribution be smaller, to more easily copy to non-networked systems. Any way to make these lighter is great, and maybe there's a better way to segment this from the operational part of the distribution?

**Comment by jgattiker** [comment]
Also, the code that validates results against matlab results are pretty uninteresting to the average user. Maybe these should be in a separate validation repo? I don't want to lose them, but I think they're pretty much mothballed now.

**Comment by granthutchings** [comment]
Yes, I think it’s right to stop tracking these.
